privaterelay/templates/base.html

```diff
@@ -33,7 +33,7 @@
     {% include "includes/modal-delete.html" %}
     {% include "includes/modal-domain-registration-confirmation.html" %}
     {% include "includes/header.html" %}
-    <firefox-private-relay-addon data-addon-installed="false"></firefox-private-relay-addon>
+    <firefox-private-relay-addon data-user-logged-in="{% with request.user.profile_set.first as user_profile %}{{ request.user.is_authenticated }}{%endwith%}" data-addon-installed="false"></firefox-private-relay-addon>
```

Comment:
This doesn't need to bother with the `profile_set` anymore:
```suggestion
    <firefox-private-relay-addon data-user-logged-in="{{ request.user.is_authenticated }}" data-addon-installed="false"></firefox-private-relay-addon>
```